src/data-unit-tests/deploy.sh

```diff
@@ -0,0 +1,10 @@
+gcloud functions deploy  data_unit_test2 \
```

Comment:
Is this file meant to be run manually or its it part of an automated pipeline. If it's meant to be automated, you may need to add the `--quiet` parameter to avoid it from expecting user input. If it's meant to be manually run, then no worries.

Reply:
It's meant to be run manually, and only if maybe one wants to make some modifications to the cloud function, however  I just added it here, so any one can know the  cloud function being used to execute the tests

Reply:
Cool, thanks